databricks/koalas/generic.py

```diff
@@ -1634,6 +1636,151 @@ def first_valid_index(self):
 
         return first_valid_idx
 
+    def filter(self, items=None, like=None, regex=None, axis=None):
+        """
+        Subset rows or columns of dataframe according to labels in
+        the specified index.
+        Note that this routine does not filter a dataframe on its
+        contents. The filter is applied to the labels of the index.
+        Parameters
```

Comment:
Ah .. I didn't notice the newlines disappeared here.. it breaks the doc rendering. Let's be careful next time.

Reply:
Oppse I'll be careful next time. I'm Sorry